common/models/access-token.js

```diff
@@ -132,26 +137,26 @@ module.exports = function(AccessToken) {
    * @param {Boolean} isValid
    */
 
-  AccessToken.prototype.validate = function(cb) {
+  AccessToken.validate = function(token, cb) {
```

Comment:
In the connector-remote ds scenario for AccessToken model the prototype method doesn't work, changing the implementation to static I was the solution that I found, I tried a couple of workarounds but nothing of them worked. I'm sure that a more elegant implementation could be done.